frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  Checkbox as ChakraCheckbox,
+  CheckboxProps as ChakraCheckboxProps,
+  Icon,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { BxCheck } from '~/assets/icons'
+import { CHECKBOX_THEME_KEY } from '~/theme/components/Field/Checkbox'
+import { FieldColorScheme } from '~/theme/foundations/colours'
+
+export interface CheckboxProps extends ChakraCheckboxProps {
+  colorScheme?: FieldColorScheme
+}
+
+export const Checkbox = ({
+  children,
+  colorScheme = 'primary',
+  ...props
+}: CheckboxProps): JSX.Element => {
+  const { icon: iconStyles } = useMultiStyleConfig(CHECKBOX_THEME_KEY, {
+    size: props.size,
+  })
+  return (
+    <ChakraCheckbox
+      icon={<Icon as={BxCheck} __css={iconStyles} />}
```

Comment:
We lose the animation because we changed the icon. Do we want to be extra and animate our checkbox icon too?

See [Chakra's implementation](https://github.com/chakra-ui/chakra-ui/blob/main/packages/checkbox/src/checkbox-icon.tsx) for reference